woocommerce/woocommerce-gutenberg-products-block
# Add loading state to shipping rates selector

Fixes #1555.

Introduce `<LoadingComponent>` which can be used to wrap components that are loading. It takes care of hiding the loading content for accessibility and decreasing its opacity.

#### Accessibility


### Screenshots

![Peek 2020-02-18 11-31](https://user-images.githubusercontent.com/3616980/74727920-2dec4700-5242-11ea-8b2c-5eac7fc30c45.gif)

### How to test the changes in this Pull Request:

1. Create a post with a _Checkout_ block.
2. Introduce and address that you know has shipping rates available.
3. Verify the shipping rates selector shows the loading spinner.
